### PR TITLE
Add base install test for TERADATA

### PR DESCRIPTION
**Pull request by mimi1vx**
basic install test for SLE-12-SP3-TERADATA

- Verification run: http://quasar.suse.cz/tests/626#

### CODE REVIEW

File: tests/qam-teradata/teradata.pm [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    change_repos_state($repos, 'disable');
+
+    install_packages($patch_infos);
+
+    change_repos_state($repos, 'enable');
+
+    zypper_call("in -l -t patch ${patches}", exitcode => [0, 102, 103], log => 'zypper.log');
+
+    prepare_system_shutdown;
+    type_string "reboot\n";
```
Comment (asmorodskyi) [inline tests/qam-teradata/teradata.pm]: do you know about `power_action` function?

File: tests/qam-teradata/teradata.pm [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    $self->wait_boot;
```
Comment (asmorodskyi) [inline tests/qam-teradata/teradata.pm]: what the point waiting for boot up if test is over ?

Comment (mimi1vx) [reply tests/qam-teradata/teradata.pm]: check if a system is an able boot after update.

File: tests/qam-teradata/teradata.pm [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    select_console 'root-console';
+
+    fully_patch_system;
+
+    set_var('MAINT_TEST_REPO', $repos);
+    add_test_repositories;
+
+    my $patches = get_patch($incident_id, $repos);
+
```
Comment (asdil12) [inline tests/qam-teradata/teradata.pm]: Maybe you could remove some of the empty lines.

**Pull request merged**